volatility3/framework/layers/vmware.py

```diff
@@ -232,6 +232,9 @@ def stack(
             )
 
             if not vmss_success and not vmsn_success:
+                vollog.warning(
+                    f"No metadata file alongside VMEM file! A VMSS or VMSN file is required to correctly process a VMEM file. These should be placed in the same directory with the same file name, e.g. sample.vmem and sample.vmsn.",
```

Comment:
I'd probably remove the exclamation mark, we're not trying to scare the user and this might be a mistake on our part (some vmem files don't strictly need an associated metadata file, but it's impossible to tell from the image alone).  I'd say rather than `file name, e.g. sample.vmem and sample.vmsn`, I'd say `basename and appropriate file extension (vmsn or vmss)`.